### DISCUSSION TITLE
Vote to retire @emmjohnson

### DISCUSSION THREAD

**Discussion by dmikusa** [paketo-buildpacks/feedback#271] [Utilities Team]
Hey https://github.com/orgs/paketo-buildpacks/teams/utilities-maintainers, I'd like to propose to move @emmjohnson to an emeritus contributor position. I do not believe they are involved in the project anymore. As is traditional, we will award the emeritus status as recognition for helping with the project.

@paketo-buildpacks/utilities-maintainers Can I get a 👍 / 👎 vote?

**Comment by ForestEckhardt** [comment]
We have reached a super majority. @emmjohnson will be awarded emeritus status.
